src/app/tests/suites/certification/Test_TC_DD_3_8.yaml

```diff
@@ -24,8 +24,162 @@ config:
 
 tests:
     - label:
-          "Repeat steps 1.a-1.o from [TC-DD-3.6]. Force exit any step in 1.a and
-          1.o."
+          "Commissioner has regulatory and fabric information available and has
+          accurate date, time and timezone"
+      verification: |
+
+      disabled: true
```

Comment:
This blank line is extremely confusing.  Why is it there?

Reply:
this Manual scripts generated using XL sheet ,there is no verification steps in that reference xl sheet so thats why blankspace is there ,let me remove this script 

https://accounts.google.com/signin/v2/identifier?service=wise&passive=1209600&continue=https%3A%2F%2Fdocs.google.com%2Fspreadsheets%2Fd%2F1h4d4J90KAxFqUL_cXZHyM3pHjHCEXzFITkQhN2KMqDY%2Fedit&followup=https%3A%2F%2Fdocs.google.com%2Fspreadsheets%2Fd%2F1h4d4J90KAxFqUL_cXZHyM3pHjHCEXzFITkQhN2KMqDY%2Fedit&ltmpl=sheets&flowName=GlifWebSignIn&flowEntry=ServiceLogin#gid=446790162